src/fidesops/ops/core/config.py

```diff
@@ -35,9 +35,10 @@ class ExecutionSettings(FidesSettings):
     """Configuration settings for execution."""
 
     privacy_request_delay_timeout: int = 3600
-    task_retry_count: int
-    task_retry_delay: int  # In seconds
-    task_retry_backoff: int
+    # By default Fidesops will not retry graph nodes
```

Comment:
Why not? Our previous defaults for this (set in the TOML) did have nonzero retries, and I think that's helpful behaviour!

Reply:
OK, I'll trust you to roll this into future improvements so that our default retry logic is reasonable. I'd argue that we should disable it in CI if we want (we can override this ourselves!) but understand if the retries are surprising right now. We can improve that in our error reporting work